platform/src/main/resources/db/migration/V1__master.sql

```diff
@@ -0,0 +1,114 @@
+CREATE SEQUENCE  IF NOT EXISTS hibernate_sequence START WITH 1 INCREMENT BY 1;
```

Comment:
🎉  I prefer this DB syntax for the migrations since it's more universally known.